docs/timeseries/index.rst

```diff
@@ -13,20 +13,22 @@ Time series (`astropy.timeseries`)
 
 .. warning::
 
-    `astropy.timeseries` is currently a work-in-progress (new in v3.2), and thus it is quite
-    possible there will be API changes in later versions of Astropy. If you have
-    specific ideas for how it might be improved, please  let us know on the
-    `astropy-dev mailing list`_ or at http://feedback.astropy.org .
+    `astropy.timeseries` is currently a work in progress (new in v3.2), and thus
+    it is possible there will be API changes in later versions of ``astropy``.
+    If you have specific ideas for how it might be improved, please let us know
+    on the `astropy-dev mailing list`_ or at http://feedback.astropy.org .
+
+.. is this warning out of date?
```

Comment:
I think it's still true, we haven't yet received as much feedback as we hoped for. 

@astrofrog can confirm it though whether he wants to keep the warning or thinks we won't change the API.

Reply:
@astrofrog do you want to keep this warning in here for now?

Reply:
Now that it's in LTS and is going to be in v4.1 too, I think we can probably remove this warning.